source/_integrations/smart_meter_texas.markdown

```diff
@@ -0,0 +1,32 @@
+---
+title: Smart Meter Texas
+description: Instructions on how to integrate Smart Meter Texas into Home Assistant.
+ha_category:
+  - Sensor
+ha_release: 0.115
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - "@grahamwetzler"
+ha_domain: smart_meter_texas
+---
+
+The [Smart Meter Texas](https://www.smartmetertexas.com/) integration allows Texas residents to integrate their electric meters into Home Assistant. This integration will create a sensor for each meter that shows the current reading.
+
+On its own this sensor is not particularly useful. However, combined with a [`utility_meter`](https://www.home-assistant.io/integrations/utility_meter/) sensor you can obtain usage over a month, for example.
+
+Any Texas electricity consumer can [register an account](https://www.smartmetertexas.com/register). To register you will need following details which can be found on your electric bill.
```

Comment:
https://www.smartmetertexas.com/register seems to redirect to /home and might change in the future.  Perhaps just go without the extended path?

Reply:
Good catch. I removed the /register link and provided better instructions to setup an account.